ccmlib/scylla_node.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, name, cluster, auto_bootstrap, thrift_interface,
         self._node_install_dir = None
         self._node_scylla_version = None
         self._relative_repos_root = None
+        self._launch_env = dict(os.environ)
```

Comment:
I'm not sure I like the idea of copying the env at class __init__,
it would block the test from doing things like change env and restart the node, as an example.

I would suggest turn it into a property function that would do the copy as needed, and add need parameters like `LD_LIBRARY_PATH` as needed

Reply:
@fruch hi Israel, thank you for your review. just wanted to understand the use case better: so it's something like

```py
node = ScyllaNode()
os.environ['FOO'] = 'bar'
node.start() # we expect $FOO to take effect here
```

right?

Reply:
i failed to foresee a use case like this but anyway. will change to a property as you suggest. personally, i am not a big fan of too many knobs.